Codeinwp/neve
# Feat: Open extra features when module is activated

### Summary

When a module is active, the extra features accordion is open by default and can be closed later by the user.

### Will affect visual aspect of the product

NO

### Screenshots 

### Test instructions


- Go to Neve options -> Neve Pro tab
- The active modules should have the extra features accordion opened by default
- When a module is switched on, the extra features accordion should open
- The accordions can be closed after that


Closes Codeinwp/neve-pro-addon#1666.
